### PR TITLE
PS-7991 Release Notes for 8.0.26-17 (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/conf.py
	new file:   source/release-notes/Percona-Server-8.0.26-17.rst
	modified:   source/release-notes/release-notes_index.rst

### CODE REVIEW

**Review by LenzGr — APPROVED** [review]
LGTM, thank you.

File: doc/source/release-notes/Percona-Server-8.0.26-17.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+The TokuDB Storage Engine was declared deprecated for Percona Server for MySQL. Starting with Percona Server 8.0.26-16, the plugins are available in the binary builds and packages but are disabled. The plugins will be removed from the binary builds and packages in a future version.
+
+New options have been added to enable the plugins if they are needed to migrate the data to another storage engine.
+
+The instructions on how to enable the plugins and more information are available at the beginning of each TokuDB topic.
+
+Bugs Fixed
+==================================================
+
+* To allow a single binary package, on platforms where ``libzbd`` is available or can be built (with 5.9 kernel and higher) ``libzbd`` statically linked into ha_rocksdb.so.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.26-17.rst]: This reads kind of funny, maybe it's just me, but wouldn't some thing like this make a little more sense?
`To allow a single binary package, statically linked libzbd into ha_rocksdb.so. Requires 5.9 kernel and higher.`

File: doc/source/release-notes/Percona-Server-8.0.26-17.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+* Fixed ZenFS issues with sysbench by using Debian 11 or the latest ``libzbd``.
+
+* Fixed a sporadic [aborting on BG write error] assertion in rocksdb.bloomfilter3.
+
+* Fixed ``WITH_ZENFS_UTILITY`` CMake option is set to ``ON`` additional logic is added to the RocksDB *CMakeLists.txt* to ensure the ``libgflags`` library is installed on the system.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.26-17.rst]:
```suggestion
* Fixed ``WITH_ZENFS_UTILITY`` CMake option when it is set to ``ON``.  Added logic to the RocksDB *CMakeLists.txt* to ensure the ``libgflags`` library is installed on the system.
```

File: doc/source/release-notes/Percona-Server-8.0.26-17.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+* Fixed when using the ``du`` system utility, the utility must be converted to computations based on the zenfs list output.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.26-17.rst]:
```suggestion
* Fixed tests that rely on the ``du`` system utility. The ``du`` utility results must be converted to computations based on the zenfs list output.
```

File: doc/source/release-notes/Percona-Server-8.0.26-17.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+* Fixed by allowing zenfs mkfs to accept an existing pre-configured aux_path.
```
Comment (george-lorch) [inline doc/source/release-notes/Percona-Server-8.0.26-17.rst]:
```suggestion
* Fixed zenfs mkfs command to allow it to accept a pre-existing aux_path. 
```

**Review by george-lorch — APPROVED** [review]

File: doc/source/release-notes/Percona-Server-8.0.26-17.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.26-17.rst]: Rewritten.

File: doc/source/release-notes/Percona-Server-8.0.26-17.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.26-17.rst]: Updated.

File: doc/source/release-notes/Percona-Server-8.0.26-17.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.26-17.rst]: Updated.

File: doc/source/release-notes/Percona-Server-8.0.26-17.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/release-notes/Percona-Server-8.0.26-17.rst]: Updated.

**Review by george-lorch — APPROVED** [review]
Looks good!

**Pull request merged**